core/controllers/base.py

```diff
@@ -440,8 +440,19 @@ def handle_exception(self, exception, unused_debug_mode):
                 in debug mode.
         """
         if isinstance(exception, self.NotLoggedInException):
-            self.redirect(
-                current_user_services.create_login_url(self.request.uri))
+
```

Comment:
Let's remove this newline; the "if" above is connected to all the logic below so it makes sense to keep them together.

Reply:
Done.